src/printer/job.js

```diff
@@ -126,3 +128,41 @@ function getFrameState(spec) {
 
   return frameState;
 }
+
+/**
+ * Returns the map canvas size in pixels based on size units and dpi given in spec
+ * @param {PrintSpec} spec
+ * @return {Array}
+ */
+function calculatePixelSize(spec) {
+  if (!spec.size[2] || spec.size[2] === 'px') {
```

Comment:
use destruct to have `dpi` and `size` const.

Reply:
not sure what you mean here

Reply:
```js
const { size, dpi } = spec
```

Reply:
ah cool wasn't aware of that, thanks